### ISSUE TITLE
zcashd page does not update info

### ISSUE THREAD

**Issue by rex4539** [adityapk00/zecwallet-electron#6]
<img width="1280" alt="Screen Shot 2020-02-03 at 19 46 46" src="https://user-images.githubusercontent.com/227442/73679872-f15a1080-46c3-11ea-9b6e-8858f6edf075.png">

## Prerequisites

- [X] Using yarn
- [X] Using node 10.x
- [X] Using an up-to-date [`master` branch](https://github.com/electron-react-boilerplate/electron-react-boilerplate/tree/master)
- [X] Using latest version of devtools. See [documentation for how to update](https://electron-react-boilerplate.js.org/docs/dev-tools)

## Expected Behavior

`zcashd` page updates info in real time.

## Current Behavior

`zcashd` page does not update info in real time.

## Steps to Reproduce (for bugs)

1. Start the app and switch to `zcashd` page.

## Context

The only way to update the page currently is to switch back and forth between tabs.

## Your Environment

- Node version : `v12.13.0`
- Version or Branch used : `master`
- Operating System and version : `macOS 10.15.3 (19D76)`

**Comment by adityapk00** [comment]
I think there is just a 20s delay, which is the rate at which it polls zcashd. Can you check if there's just a delay or whether it's not actually refreshing it at all.

**Comment by rex4539** [comment]
I waited several minutes. It's just not updating.

**Comment by adityapk00** [comment]
Cool, thanks. I'll look into it.

**Comment by adityapk00** [comment]
Oh, you know what - Do you have a lot of transactions and/or addresses in this zcashd wallet? I think this bug is probably because of another bug where too many transactions are slowing down the refresh. Also probably related to #5

**Comment by rex4539** [comment]
This wallet shows 9 shielded and 100 transparent addresses, out of which only 1 shielded address has fund in it.

96 transactions in total.

**Comment by adityapk00** [comment]
I think for some reason the updates are not happening in your app. Can you try this:

Start the app with `yarn dev`, and in the console, can you check for errors? Are you seeing messages like `All done, setting up next fetch` (which indicates that the refresh successfully happened)

**Comment by rex4539** [comment]
I see this.
```
Uncaught (in promise) Error: ENOENT: no such file or directory, open '/Users/rex/Library/Application Support/Zcash/senttxstore.dat'
  | async function (async) |   |  
-- | -- | -- | --
  | refresh | @ | rpc.js:132
  | (anonymous) | @ | rpc.js:77
  | setTimeout (async) |   |  
  | configure | @ | rpc.js:77
  | RouteApp.setRPCConfig | @ | Routes.js:179
  | getInfo | @ | LoadingScreen.js:133
  | async function (async) |   |  
  | getInfo | @ | LoadingScreen.js:128
  | setupNextGetInfo | @ | react-hot-loader.development.js:714
  | (anonymous) | @ | LoadingScreen.js:120
  | setTimeout (async) |   |  
  | setupNextGetInfo | @ | LoadingScreen.js:120
  | setupNextGetInfo | @ | react-hot-loader.development.js:714
  | (anonymous) | @ | LoadingScreen.js:81
  | async function (async) |   |  
  | (anonymous) | @ | LoadingScreen.js:66
  | componentDidMount | @ | LoadingScreen.js:63
  | componentDidMount | @ | react-hot-loader.development.js:704
  | commitLifeCycles | @ | react-dom.development.js:22067
  | commitLayoutEffects | @ | react-dom.development.js:25301
  | callCallback | @ | react-dom.development.js:337
  | invokeGuardedCallbackDev | @ | react-dom.development.js:386
  | invokeGuardedCallback | @ | react-dom.development.js:441
  | commitRootImpl | @ | react-dom.development.js:25039
  | unstable_runWithPriority | @ | scheduler.development.js:818
  | runWithPriority$2 | @ | react-dom.development.js:12131
  | commitRoot | @ | react-dom.development.js:24888
  | finishSyncRender | @ | react-dom.development.js:24295
  | performSyncWorkOnRoot | @ | react-dom.development.js:24273
  | scheduleUpdateOnFiber | @ | react-dom.development.js:23664
  | updateContainer | @ | react-dom.development.js:27060
  | (anonymous) | @ | react-dom.development.js:27484
  | unbatchedUpdates | @ | react-dom.development.js:24399
  | legacyRenderSubtreeIntoContainer | @ | react-dom.development.js:27483
  | render | @ | react-dom.development.js:27620
  | (anonymous) | @ | index.js:9
  | ./app/index.js | @ | renderer.dev.js:6223
  | __webpack_require__ | @ | bootstrap:726
  | fn | @ | bootstrap:100
  | 0 | @ | only-dev-server.js:102
  | __webpack_require__ | @ | bootstrap:726
  | (anonymous) | @ | bootstrap:793
  | (anonymous) | @ | bootstrap:793
```

**Comment by adityapk00** [comment]
Fixed in bf89736
